app/serializers/concerns/optional_serialized_attributes.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+module OptionalSerializedAttributes
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def attribute_if_positive(optional_attribute)
+      attribute optional_attribute, if: -> { object.send(optional_attribute).positive? }
+    end
```

Comment:
I disagree. This was done advisedly. I fail to see how sending bloated responses to clients would help anyone.